packages/expo-router/src/utils/splash.ts

```diff
@@ -0,0 +1,39 @@
+import { requireOptionalNativeModule } from 'expo';
+
+const SplashModule = requireOptionalNativeModule('ExpoSplashScreen');
```

Comment:
If expo router requires the module like this and not from `expo-splash-screen` maybe we can remove the private functions from the public api?

Reply:
done